CHANGELOG-0.2.0.md

```diff
@@ -0,0 +1,21 @@
+# v0.2.0
+
+
+# Changelog since v0.1.0
```

Comment:
This should go after support status, and then everything nested one more level

Reply:
fixed